cowdao_cowpy/order_book/generated/model.py

```diff
@@ -592,12 +594,14 @@ class OrderQuoteRequest(BaseModel):
     priceQuality: Optional[PriceQuality] = "verified"
     signingScheme: Optional[SigningScheme] = "eip712"
     onchainOrder: Optional[Any] = Field(
-        False,
+        default=False,
         description='Flag to signal whether the order is intended for on-chain order placement. Only valid\nfor non ECDSA-signed orders."\n',
     )
 
 
 class OrderQuoteResponse(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+
     quote: OrderParameters
     from_: Optional[Address] = Field(None, alias="from")
```

Comment:
`#question` just saw [this](https://github.com/pydantic/pydantic/issues/5893) discussion in the Pydantic repo. I assume you might've seen it, but have you tried doing:

```suggestion
    from_: Annotated[Optional[Address], Field(None, alias=str('from'))]
```

Reply:
Sorry for late response, I needed a sick day. I actually missed this! However, for some reason it triggers many others problems. This is the state now:

```
peter/byna,me* $ poetry run pyright .                                                                                                                            [10:43:03]
0 errors, 0 warnings, 0 information
```

and if I change to this Annotated-pattern:

```
peter/byna,me* $ poetry run pyright .                                                                                                                            [10:44:11]
/Users/peter/Dropbox/Projects/Gnosis/cow-py/cowdao_cowpy/cow/swap.py
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/cowdao_cowpy/cow/swap.py:47:19 - error: Argument of type "ChecksumAddress" cannot be assigned to parameter "sellToken" of type "Address" in function "__init__"
    "ChecksumAddress" is not assignable to "Address" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/cowdao_cowpy/cow/swap.py:48:18 - error: Argument of type "ChecksumAddress" cannot be assigned to parameter "buyToken" of type "Address" in function "__init__"
    "ChecksumAddress" is not assignable to "Address" (reportArgumentType)
/Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/e2e/test_post_and_cancel_order_live_e2e.py
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/e2e/test_post_and_cancel_order_live_e2e.py:60:19 - error: Argument of type "Literal['0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d']" cannot be assigned to parameter "sellToken" of type "Address" in function "__init__"
    "Literal['0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d']" is not assignable to "Address" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/e2e/test_post_and_cancel_order_live_e2e.py:61:18 - error: Argument of type "Literal['0x177127622c4A00F3d409B75571e12cB3c8973d3c']" cannot be assigned to parameter "buyToken" of type "Address" in function "__init__"
    "Literal['0x177127622c4A00F3d409B75571e12cB3c8973d3c']" is not assignable to "Address" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/e2e/test_post_and_cancel_order_live_e2e.py:62:18 - error: Argument of type "Address" cannot be assigned to parameter "receiver" of type "Address | None" in function "__init__"
    Type "Address" is not assignable to type "Address | None"
      "eth_typing.evm.Address" is not assignable to "cowdao_cowpy.order_book.generated.model.Address"
      "Address" is not assignable to "None" (reportArgumentType)
/Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:68:19 - error: Argument of type "Literal['0x']" cannot be assigned to parameter "sellToken" of type "Address" in function "__init__"
    "Literal['0x']" is not assignable to "Address" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:69:18 - error: Argument of type "Literal['0x']" cannot be assigned to parameter "buyToken" of type "Address" in function "__init__"
    "Literal['0x']" is not assignable to "Address" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:70:18 - error: Argument of type "Literal['0x']" cannot be assigned to parameter "receiver" of type "Address | None" in function "__init__"
    Type "Literal['0x']" is not assignable to type "Address | None"
      "Literal['0x']" is not assignable to "Address"
      "Literal['0x']" is not assignable to "None" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:71:17 - error: Argument of type "Literal['app_data_object']" cannot be assigned to parameter "appData" of type "AppData | AppDataHash | None" in function "__init__"
    Type "Literal['app_data_object']" is not assignable to type "AppData | AppDataHash | None"
      "Literal['app_data_object']" is not assignable to "AppData"
      "Literal['app_data_object']" is not assignable to "AppDataHash"
      "Literal['app_data_object']" is not assignable to "None" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:72:21 - error: Argument of type "Literal['0x']" cannot be assigned to parameter "appDataHash" of type "AppDataHash | None" in function "__init__"
    Type "Literal['0x']" is not assignable to type "AppDataHash | None"
      "Literal['0x']" is not assignable to "AppDataHash"
      "Literal['0x']" is not assignable to "None" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:74:22 - error: Argument of type "Literal['verified']" cannot be assigned to parameter "priceQuality" of type "PriceQuality | None" in function "__init__"
    Type "Literal['verified']" is not assignable to type "PriceQuality | None"
      "Literal['verified']" is not assignable to "PriceQuality"
      "Literal['verified']" is not assignable to "None" (reportArgumentType)
  /Users/peter/Dropbox/Projects/Gnosis/cow-py/tests/order_book/test_api.py:75:23 - error: Argument of type "Literal['eip712']" cannot be assigned to parameter "signingScheme" of type "SigningScheme | None" in function "__init__"
    Type "Literal['eip712']" is not assignable to type "SigningScheme | None"
      "Literal['eip712']" is not assignable to "SigningScheme"
      "Literal['eip712']" is not assignable to "None" (reportArgumentType)
12 errors, 0 warnings, 0 information
```

And actually I'm not sure why these errors don't complain even now, as they seem legit 🤷 

Do you mind going in with this PR, and taking a high level view on typing in another PR? (I can help once I have some time between Gnosis tasks)

Reply:
interesting! No worries, I think that's indeed ooo for this PR. We can open an issue to address that specific item.